soc/arm/nordic_nrf/nrf91/Kconfig.defconfig.nrf9160_SICA

```diff
@@ -0,0 +1,20 @@
+# Kconfig.defconfig.nrf9160 - Nordic Semiconductor nRF9160 MCU
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+
+if SOC_NRF9160_SICA
+
+config SOC
+	string
+	default "nRF9160_SICA"
```

Comment:
Why "SICA"?

Reply:
@pizi-nordic  This is the code-name for current flavor of nrf9160

Reply:
See the product options of nRF9160: https://www.nordicsemi.com/DocLib/Content/Product_Spec/nRF9160/latest/nRF9160_html5_keyfeatures